db/migrate/20210806011757_create_topics.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTopics < ActiveRecord::Migration[6.1]
+  def change
+    create_table :topics do |t|
+      t.string :name
```

Comment:
Add this as you have the validation in the model. Then you can create a test for validation.

```suggestion
      t.string :name, null: false
```